logstash-core/lib/logstash/agent.rb

```diff
@@ -34,6 +34,8 @@ def initialize(params)
     @pipelines = {}
     @started_at = Time.now
     @node_name = params[:node_name] || Socket.gethostname
+    @web_api_http_port = params[:web_api_http_port] || 9600
```

Comment:
I think the default value should be handled inside the webservice class, It's not a concern of the agent to setup the default values for the port. Also moving it there align with our discussion from yesterday concerning a runner class.